### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by ben-d-marks**
New Branch name. Apparently got rid of all my shiny comments. Will repopulate

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/lib/node_modules
+/lib/.sass-cache
+*.css.map
+*.css
+
```
Comment (ben-d-marks) [inline .gitignore:5]: **Changed .gitignore**
reflected collaborator (developer) push as opposed to user push

File: data/inventory.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    "make": "Tesla",
+    "model": "3",
+    "color": "Excelsion",
+    "interiorColor": "Roomy Grey",
+    "interiorMaterial": "Impervium",
+    "price": 42845,
+    "mileage": 120,
+    "inventoryNum": "t32018ei01",
+    "description": "This is a brnd new TeslaIII... Or 3.... or Three. 'In Soviet Russia, car drives YOU!' Git you one!"
+  }, {
```
Comment (ben-d-marks) [inline data/inventory.json:68]: **Added more cars**
for row creation demonstration purposes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <title>Buy a Car!</title>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.5/css/bootstrap.min.css" integrity="sha384-AysaV+vQoT3kOAXZkl02PThvDr8HYKPZhNT5h/CXfBThSRXQ6jW5DO2ekP5ViFdi" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header class="page-header container-fluid">
+<!-- 4. Put a standard Bootstrap navbar element at the top of your page. -->
+      <nav class="navbar navbar-light bg-faded row">
+        <div class="col-xs-12">
```
Comment (ben-d-marks) [inline index.html:17]: **fix Navbar stacking weirdness**
rearranged elements to make mar pretty on narrow views
switched all columns to xs in an effort to make pretty

File: javascripts/activateEvents.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        for (let j = 0; j < cards.length; j++){
+          CarLot.resetCard(cards[j]);
+        }
+        CarLot.formatCard(cards[i], "#a2d2d2");
+      });
+    }
+
+    document.getElementById("descText").addEventListener("keyup", function(){
+      if (!editTarget){
+        alert("Please choose a car...");
```
Comment (ben-d-marks) [inline javascripts/activateEvents.js:25]: **No edit before selection**
Addressed no item selected "feature"
no error thrown now

Comment (ben-d-marks) [inline javascripts/activateEvents.js:25]: **No edit before selection**
Addressed no item selected "feature"
no error thrown now

File: javascripts/quiz.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  // Loop over the inventory and populate the page
+  var tempAddition = '';
+  for (let i = 0; i < inventory.length; i++){
+    if (i % 3 === 0){
+      tempAddition += '<div class="row">';
+    } 
+    tempAddition += '<div class="col-xs-4 ' + i + '" id="';
+    tempAddition += inventory[i].inventoryNum + i;
+    tempAddition += '"><div class="car-card col-xs-12"><div class="panel panel-default">';
+    tempAddition += '<div class="panel-heading"><h3 class="panel-title">';
```
Comment (ben-d-marks) [inline javascripts/quiz.js:18]: **fix layout claustrophobia**
added a div to my div in order to take advantage of Bootstrap's native margins

File: lib/Gruntfile.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+module.exports = function(grunt) {
```
Comment (ben-d-marks) [inline lib/Gruntfile.js:1]: **supplied gruntfile.js**
self-explanatory

File: lib/package.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (ben-d-marks) [inline lib/package.json:1]: **supplied package.json**
self-explanatory

File: sass/_normalize.scss [hunk 8]
```diff
@@ -0,0 +1,419 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (ben-d-marks) [inline sass/_normalize.scss:1]: **included .scss files***
included as part of my template.scss chain

File: sass/_template.scss [hunk 9]
```diff
@@ -0,0 +1,78 @@
+// Include Normalize in the final CSS
```
Comment (ben-d-marks) [inline sass/_template.scss:1]: **included .scss files***
included as part of my template.scss chain

File: sass/main.scss [hunk 10]
```diff
@@ -0,0 +1,55 @@
+@import 'template';
```
Comment (ben-d-marks) [inline sass/main.scss:1]: **included .scss files***
I should be utilizing sass formatting a bit more

**Review by ben-d-marks — COMMENTED** [review]
Overall: pretty "OK".
😐

**Comment by JoeShep** [comment]
👍 Merge it!

**Pull request merged**